BLSQ/iaso
# IA-3410: Rename permission "Submissions - Read Only" to "Forms and submissions - Read only" and org unit permissions

Explain what problem this PR is resolving
- Rename permission "Submissions - Read Only" to "Forms and submissions - Read only" and org unit permissions
Related JIRA tickets : [IA-3410](https://bluesquare.atlassian.net/browse/IA-3410)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Rename read only and read and write permission for Submissions and Org Units

## How to test

- Edit or Create a user(or userrole)
- Check org units and submission permission if they'v been renamed

## Print screen / video
![Screenshot-from-2024-09-12-11-42-23](https://github.com/user-attachments/assets/b0a32281-23d9-4eeb-bce6-a440984c3a46)

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3410]: https://bluesquare.atlassian.net/browse/IA-3410?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
